### PR TITLE
Automate pac4j client config

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -50,7 +50,7 @@ jodaTimeVersion=2.9
 reflectionsVersion=0.9.10
 guavaVersion=18.0
 
-pac4jVersion=1.8.0
+pac4jVersion=1.8.1-SNAPSHOT
```
Comment (leleuj) [inline gradle.properties]: pac4j v1.8.1 will be released end November / begin December. I'll update the version after that.

File: cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp [hunk 2]
```diff
@@ -109,6 +109,19 @@
     <div class="sidebar-content">
         <p><spring:message code="screen.welcome.security" /></p>
 
+        <c:if test="${!empty pac4jUrls}">
+            <div id="list-providers">
+                <h3><spring:message code="screen.welcome.label.loginwith" /></h3>
+                <form>
+                    <ul>
```
Comment (mmoayyed) [inline cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp]: Are these urls that would be decorated with logo images of various providers? don't see the images in PR :)

Comment (leleuj) [inline cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp]: Good point. I finally used the name of the providers as I encountered two problems: 30 images at least would have been required to match all available providers and I wasn't sure to know what would make sense for SAML, OpenID Connect, CAS OAuth wrapped server...

Comment (mmoayyed) [inline cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp]: No problem. 

I was thinking to modify the demo server to include this configuration by default, and then thought images would look good there for the demo but that's a customization we can make to the demo server. No big deal.

Comment (leleuj) [inline cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp]: It makes life easier for CAS deployers: as soon as they add a new provider, it shows up. But I doubt that any CAS deployer will keep the layout and design on these new links (and the login page more generally).

Comment (leleuj) [inline cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp]: BTW, I forgot to update the documentation. Will add that shortly.

**Pull request merged**